web/static/js/components/remote_retro.jsx

```diff
@@ -23,6 +22,7 @@ export class RemoteRetro extends Component {
     retroChannel.join()
       .receive("ok", retroState => {
         this.setState(retroState)
+        actions.updateStage(retroState.stage)
```

Comment:
Here, it's too early to remove the `setState` invocation, as we're still relying on the retro's `inserted_at` timestamp.